DigitalLearningSolutions.Data/Services/UserDataService.cs

```diff
@@ -70,9 +70,17 @@ public UserDataService(IDbConnection connection)
                         cd.LastName,
                         cd.Password,
                         cd.Approved,
-                        cd.ProfileImage
+                        cd.ProfileImage,
+                        cd.Answer1,
+                        cd.Answer2,
+                        cd.Answer3,
+                        cd.Answer4,
+                        cd.Answer5,
+                        cd.Answer6,
+                        jg.JobGroupName
```

Comment:
I noticed that as part of the registration journey we now have an IJobGroupsService method string? GetJobGroupName(int jobGroupId);
Should I be using this and only querying the cd.JobGroupId here rather than just joining on the table?

Reply:
I want to say that it is fine to grab the name as part of this query, as it is only grabbing the name from an Id that is already stored on the record. If we wanted to grab more details from JobGroups, then we should almost certainly call the other service. It does mean that the GetJobGroupName method is slightly less useful than it could be, but I'm leaning toward having a more redundant method that is only used in specific circumstances and having less calls to the services/DB for something this simple. Though @stellake  might have other ideas and I am happy to be overruled.

Reply:
I think it makes sense to join here. 👍